rotkehlchen/db/filtering.py

```diff
@@ -441,19 +443,29 @@ def prepare(self) -> Tuple[List[str], List[Any]]:
 
 
 @dataclass(init=True, repr=True, eq=True, order=False, unsafe_hash=False, frozen=False)
-class DBMultiStringFilter(DBFilter):
+class DBMultiValueFilter(Generic[T], DBFilter):
     """Filter a column having a string value out of a selection of values"""
     column: str
-    values: List[str]
+    values: List[T]
     operator: str = 'IN'
 
-    def prepare(self) -> Tuple[List[str], List[Any]]:
+    def prepare(self) -> Tuple[List[str], List[T]]:
         return (
             [f'{self.column} {self.operator} ({", ".join(["?"] * len(self.values))}) OR {self.column} IS NULL'],  # noqa: E501
             self.values,
         )
 
 
+class DBMultiStringFilter(DBMultiValueFilter):
+    """Filter a column having a string value out of a selection of values"""
+    values: List[str]
```

Comment:
```suggestion
class DBMultiStringFilter(DBMultiValueFilter[str]):
    """Filter a column having a string value out of a selection of values"""
    ....
```

Or you can just use DBMultiValueFilter[str] everywhere. Up to you. It's mostly readability.